self-api/src/main/java/com/selfxdsd/api/ApiToken.java

```diff
@@ -0,0 +1,29 @@
+package com.selfxdsd.api;
```

Comment:
@andreoss Please add the Licence to all the new Java files (just copy it from an existing file).

Reply:
@amihaiemil Done.